src/Foundation/NSSecureCoding.cs

```diff
@@ -19,8 +21,8 @@ public partial class NSSecureCoding {
 
 		public static bool SupportsSecureCoding (Type type)
 		{
-			if (type == null)
-				throw new ArgumentNullException ("type");
+			if (type is null)
+				throw new ArgumentNullException (nameof (type));
```

Comment:
Better use `ObjCRuntime.ThrowHelper.ThrowArgumentNullException ((nameof (type));`
Throwing an exception generally means the method can't be inlined.

Reply:
Fixed.